src/vs/workbench/services/editor/common/editorGroupFinder.ts

```diff
@@ -205,6 +206,9 @@ function matchesEditor(typedEditor: EditorInput, editor: EditorInput | IUntypedE
 	// editors that have no `override` defined.
 	//
 	// TODO@lramos15 https://github.com/microsoft/vscode/issues/131619
+	if (typedEditor instanceof DiffEditorInput && isResourceDiffEditorInput(editor)) {
```

Comment:
@bpasero FYI `revealIfVisible` and `revealIfOpened` were broken with untyped inputs due to doing a top level resource check only which diff editor inputs don't have.

@mjbvz This is yet another reason we hope #131619 can eventually be revisited and addressed. Webviews needing to know the group for the input is a very fragile check on just the resource as we don't know the editor type at this time

Reply:
Is it possible to have a unit test for this?

Btw could this apply to side by side editors as well, which are not diff inputs but behave very similarly?